pkgs/sqlc-dev/sqlc/registry.yaml

```diff
@@ -1,10 +1,11 @@
 packages:
   - type: github_release
-    repo_owner: kyleconroy
+    repo_owner: sqlc-dev
     repo_name: sqlc
```

Comment:
Let's set `aliases` to keep the compatibility.

https://aquaproj.github.io/docs/reference/registry-config/aliases/

Reply:
b8444769890663051990a3d1c974136c9b936910